sql/2019/10_SEO/10_15.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+
+# speed metrics (FCP, server response time)
```

Comment:
One idea would be to query the CrUX dataset (which shares the same URLs) for the % of websites with slow FCP (>= 10%, per PageSpeed Insights)


```sql
#standardSQL
SELECT
  ROUND(COUNTIF(slow_fcp >= 0.10) * 100 / COUNT(0), 2) AS pct_slow_fcp
FROM
  `chrome-ux-report.materialized.metrics_summary`
```

Reply:
I tried to adept to this dataset, but not very familiar with the 10% and slow_x metrics (and densities). May need some extra eyes/help.